src/test/java/LoginTests.java

```diff
@@ -22,20 +22,20 @@ public void LoginValidEmailPasswordTest () {
         LoginPage loginPage = new LoginPage(driver);
         HomePage homePage = new HomePage(driver);
 
-        loginPage.provideEmail("demo@class.com");
-        loginPage.providePassword("te$t$tudent");
-        loginPage.clickSubmitBtn();
+        loginPage.provideEmail("demo@class.com")
+                .providePassword("te$t$tudent")
+                .clickSubmitBtn();
+
+
 
         Assert.assertTrue(homePage.getUserAvatar().isDisplayed());
 
     }
 
     @Test(enabled = true, priority = 2, description = "LoginInvalidEmailValidPasswordTest")
     public static void loginInvalidEmailValidPasswordTest () {
+        login("invalid@class.com", "te$t$tudent");
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example `public void loginValidEmailPasswordTest`